github/orgs_audit_log.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// GetAuditLogOptions sets up optional parameters to query audit-log endpoint.
+type GetAuditLogOptions struct {
```

Comment:
@gmlewis Would you recommend that I move `After` and `Before` to `ListCursorOptions` 
IOW
```
type GetAuditLogOptions struct {
	Phrase  *string `json:"phrase,omitempty"`   // A search phrase. (Optional.)
	Include *string `json:"include,omitempty"`  // Event type includes. Can be one of "web", "git", "all". Default: "web". (Optional.)
	Order   *string `json:"order,omitempty"`    // The order of audit log events. Can be one of "asc" or "desc". Default: "desc". (Optional.)

         // Pagination Options
	ListCursorOptions
}
```

and update 
```
// ListCursorOptions specifies the optional parameters to various List methods that
// support cursor pagination.
type ListCursorOptions struct {
	// For paginated result sets, page of results to retrieve.
	Page string `url:"page,omitempty"`

	// For paginated result sets, the number of results to include per page.
	PerPage int `url:"per_page,omitempty"`
        
        // A cursor, as given in the Link header. If specified, the query only searches for events after this cursor. (Optional.)
	After   string `url:"after,omitempty"`    

        // A cursor, as given in the Link header. If specified, the query only searches for events before this cursor. (Optional.)
	Before  string `url:"before,omitempty"`   

}
```

Reply:
@sridharavinash - in the past, I think we decided that the `After` and `Before` URLs were not useful to users of this client library, so we have explicitly not added them.

So unless you have any objections, I think we can just drop them.

Reply:
Unfortunately for the AuditLog API we don't have a `page` param, we exclusively use `before` / `after` link `rel` headers to navigate pages. Using the API call with `PerPage` will probably be limited to just viewing one page without any forward/backwards navigation of results. 

I can drop it if that's preferred, it can always be added if someone needs it.

Reply:
Ah!  OK, @sridharavinash - then in that case if it is going to be useful, go ahead and add it.

Now the question is whether or not it belongs in `ListCursorOptions`, and I'm guessing the answer is "yes", so go ahead and include it.

Reply:
added in https://github.com/google/go-github/pull/1820/commits/1814994b6d8b1ed35ae0ae09b6a5e34418c8f2f1